CHANGELOG.md

```diff
@@ -13,8 +11,24 @@
 
 ### Security
 
-## [Unreleased]
+## [0.12.0] - 2021-05-19
+### Added
+- Add attr_accessor for `service` to class `Shipcloud::Shipment` to be able to access the service attribute at the shipment object.
```

Comment:
Please add the changes only in the "Unreleased" section.
The assignment to a version will take place later.